docs/release_guide.md

```diff
@@ -54,28 +77,17 @@ If there are other commits on the `main` branch to be included in the release (s
 
 Create a new commit on the patch branch to update the hardcoded version information in the following locations:
 
-* The container image tag in [charts/osm/values.yaml](/charts/osm/values.yaml)
-* The chart and app version in [charts/osm/Chart.yaml](/charts/osm/Chart.yaml)
-* The default osm image tag in [osm cli mesh upgrade](/cmd/cli/mesh_upgrade.go)
+* The control plane image digests  defined by `OpenServiceMesh.image.digest` for images in [charts/osm/values.yaml](/charts/osm/values.yaml) from the image digests obtained from the Pre-release workflow. For example, if the osm-controller image digest is `sha256:eb194138abddbe271d42b290489917168a6a891a3dabb575de02c53f13879bee`, update the value of `OpenServiceMesh.image.digest.osmController` to `sha256:eb194138abddbe271d42b290489917168a6a891a3dabb575de02c53f13879bee`.
+* Replace the `latest-main` tag with the release version tag for all images throughout the repo, e.g. `v0.4.0`.
```

Comment:
Does this mean the places that were listed here before, like docs/example/manifests/apps/bookbuyer.yaml? I know it was really verbose and needed to be changed fairly often, but I appreciated it when it came time to make a release since it's not as easy as a global find/replace across the repo.

Even if we don't want a super verbose list, even mentioning the general locations where these tags exist would be helpful, like docs/example/manifests.

Reply:
Added a note regarding the locations where this update is applicable.